vignettes/articles/installing-cloudvolume-meshparty.Rmd

```diff
@@ -154,8 +154,8 @@ neuron_skel <- read.neuron.swc(sprintf("flywire_skeletons/%s.swc",id))
 
 nopen3d()
 plot3d(FAFB, alpha = 0.1)
-plot3d(neuron_skel, WithNodes = F, col = "red", lwd = 2)
-wire3d(neuron_mesh, col = "cyan", alpha = 0.2)
+plot3d(neuron_skel*1000, WithNodes = F, col = "red", lwd = 2)
```

Comment:
Just a note that meshparty is the odd one out here in turning the skeletons to µm